pkgs/stdenv/generic/setup.sh

```diff
@@ -395,7 +395,7 @@ findInputs() {
     local -i relHostOffset
```

Comment:
You wanted this `-i` removed, right?

Reply:
Thank you, you're right, I forgot that one.

Reply:
I just checked this again. I tried separating the PRs as much as possible and some of the other `-i` are taken care of in another PR. I'm going to remove that one as well though.
The original intention was to make it easier to test. But since the hydra tests are so long, they have had to be grouped anyways. Now I realise it wasn't the best strategy.

Reply:
I noticed this declaration because you did update the variable's use.

Reply:
Yes, you're right on this one.